webclient/src/containers/Login/Login.tsx

```diff
@@ -223,7 +224,7 @@ const Login = ({ state, description }: LoginProps) => {
   };
 
   return (
-    <div className={'login overflow-scroll ' + classes.root}>
+    <Layout className={'login overflow-scroll ' + classes.root} showNav={false}>
```

Comment:
I like where this is going, might prefer the component be called MainNavigation or something to that effect.

Reply:
Can you expand on your concerns?  My mental model is that changing routes will generate a fresh component tree with its own periods for the layout component.  There shouldn't be any concern for stake components hanging around.

Reply:
Lets have a quick chat about it tomorrow or this weekend, but dont treat it as a blocker.

Reply:
Following up, we chatted.  Seavor's concern is valid in certain contexts, but not the way this was implemented.